pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.poetry]
+name = "moviecenter"
+version = "0.1.0"
+description = ""
+authors = ["Eli"]
+
+[tool.poetry.dependencies]
+python = "3.10"
+pydantic = "1.10.2"
```

Comment:
Why are you pinning the version in this file? That's the job of the lock file.
In the `pyproject.toml` you specify version constraints, so don't pin unless you only work with this version.
Try pinning only the major+minor, not the patch

Reply:
Old habit. I remember doing this so that no one will update reps by mistake and break something. But I guess this should be resolved with proper CI.